jdaviz/configs/imviz/tests/test_astrowidgets_api.py

```diff
@@ -320,7 +320,7 @@ def test_mvp_markers(self):
         assert self.imviz.app.data_collection.labels == [
             'has_wcs[SCI,1]', 'no_wcs[SCI,1]']
 
-        assert len(self.imviz.app.data_collection.links) == 10
+        assert len(self.imviz.app.data_collection.links) == 2
```

Comment:
That is quite the change. What are the consequences?

Reply:
Most of those 10 links were duplicates from what I can tell, so there shouldn't be any negative consequences.

Reply:
This could be _huge_ for performance 🤩 !  Hopefully other tests would catch any unforeseen issues, but probably would be prudent to manually test as well (before and after merge).

Reply:
Should we test also the properties of the links? Maybe CIDs + labels?